packages/domain-picker/tsconfig.json

```
@@ -0,0 +1,35 @@
+{
+	"compilerOptions": {
+		"target": "ES5",
+		"lib": [ "DOM", "ES2015" ],
+
+		"baseUrl": ".",
+		"module": "esnext",
+		"allowJs": false,
+		"jsx": "react",
+		"declaration": true,
+		"declarationDir": "dist/types",
+		"outDir": "dist/esm",
+		"rootDir": "src",
+
+		"strict": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"noImplicitReturns": true,
+		"noFallthroughCasesInSwitch": true,
+
+		"moduleResolution": "node",
+		"esModuleInterop": true,
+
+		"forceConsistentCasingInFileNames": true,
+
+		"typeRoots": [ "../../node_modules/@types" ],
+		"types": [],
```

Related to an `import type` suggestion, I'm curious to try this configuration:

```suggestion
		"types": [],

		"importsNotUsedAsValues": "error",
```

https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/#type-only-imports-exports

---

Thanks! This is good. https://github.com/Automattic/wp-calypso/pull/42409/commits/8dc117e6c1903a545eb66bb404020500817d5b3a